tensorboard/main.py

```diff
@@ -48,6 +48,7 @@
 
 from tensorboard import default
 from tensorboard import program
+from tensorboard.plugins import base_plugin
```

Comment:
Please add the corresponding Bazel dependency: `//tensorboard:main`
should depend on `//tensorboard/plugins:base_plugin`.